mitodl/mitxpro
# feat: enterprise page

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/hq/issues/1370

#### What's this PR do?
Adds an enterprise page in CMS


#### How should this be manually tested?

- Open the terminal and run the `configure_wagtail` command.
- After that, go to the `/enterprise` URL to check if the page is visible.
- Add the remaining sections in the CMS.
- Compare the design of the page with the one in the Figma [design](https://www.figma.com/proto/zxokJkYicrTlgUM5a3dlHY/xPro-%7C-Marketing-Page?node-id=110-115&starting-point-node-id=110%3A115&mode=design&t=qVuSXRuEe6CkilCN-1).
- Check if all the links used in the page are correct.
- Verify the form on the page.
- Test the responsiveness of the page on mobile screens using the Figma [link](https://www.figma.com/proto/zxokJkYicrTlgUM5a3dlHY/xPro-%7C-Marketing-Page?page-id=482%3A67&type=design&node-id=482-398&viewport=1736%2C1355%2C0.8&t=9C36Voe9dA3a0tez-1&scaling=min-zoom&mode=design).

#### Screenshots (if appropriate)
(Desktop)
![Enterprise Page Desktop SS](https://github.com/mitodl/mitxpro/assets/32649010/1ef09f63-de4f-44a7-bbec-f49af46f1650)

(Mobile Screen)

![Enterprise-Page-Mobile-SS](https://github.com/mitodl/mitxpro/assets/32649010/cc25489d-3b4d-4294-a5b0-3f12623d6423)


<img width="390" alt="Screenshot 2023-12-12 at 3 32 10 AM" src="https://github.com/mitodl/mitxpro/assets/32649010/6307a41d-3af8-478f-b2b6-3aafc9075851">
